airflow/config_templates/default_airflow.cfg

```diff
@@ -256,6 +256,15 @@ dag_processor_manager_log_location = {AIRFLOW_HOME}/logs/dag_processor_manager/d
 # Default to use task handler.
 task_log_reader = task
 
+[secrets_backend]
+# Name of secrets backend to enable (will precede env vars and metastore in search path)
+class_name =
```

Comment:
hmm, I think we might be better off with a different name, like "secrets_client"  or "class_path" maybe?

It is not just a "name", it would need entire path. WDYT?

https://airflow.apache.org/docs/stable/configurations-ref.html#api-client

Reply:
you could say it is the "fully qualified" name :)
class path sounds like java to me...

there is another example -- ([api]/auth backend)[https://airflow.apache.org/docs/stable/configurations-ref.html#api].

we could call this key "secrets_backend", like is done with , and change the section name to `[secrets]`?  WDYT?
so like this:
```
[secrets]
secrets_backend = 
```

which is better?  too many possibilities!

```
[secrets_backend]
secrets_client = 
client_kwargs = 
```
```
[secrets] # or [secrets_backend]

# choose one from these
secrets_backend = 
class_name = 
class_path

# choose one from these:
secrets_backend_kwargs = 
backend_kwargs = 
config_kwargs = 
client_kwargs = 
client_json =
```

any proposal?

Reply:
Wow 🗡 😄 

```
[secrets]
backend = 
backend_kwargs = 
```
OR
```
[secrets]
client =
client_kwargs = 
```

I will let you select 1 from the above two options :D I will be happy with whatever you select

Reply:
this is tough!  i think both are good.  @potiuk ?

Reply:
maybe backend since that's referenced in base class name?

Reply:
(although i can change base class name to Client also... not to open another can of worms or anything...)

Reply:
i'll go with backend for now... :)